code-formatter/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM ruby:2.6-alpine
+FROM ruby:3.1.2-alpine3.15
+
+ENV TERRAFORM_VERSION=1.1.9
```

Comment:
@NickWalt01 , this may be an issue as we havent moved to `tf 1`. Can u please create a release with the current main branch just in case we need to refer an older version.

Reply:
Hello @poornima-krishnasamy a new release has been created: https://github.com/ministryofjustice/github-actions/releases/tag/v3

And thanks for approving these PR's.